.github/workflows/tests.yml

```diff
@@ -134,10 +143,17 @@ jobs:
       arch: ${{ matrix.arch }}
       flavor: ${{ matrix.flavor }}
   test_flavors_qemu:
-    needs: intersect_matrices
+    needs: [ intersect_matrices, test_flavors_chrooted ]
```

Comment:
I'm not sure I see the advantage in waiting for the chroot tests before running the QEMU test.
Both are running on free github runners, so unlike with the platform tests we do not safe any money from skipping the QEMU tests. However, enforcing an ordering here will mean that the total runtime is going to suffer due to decreased parallelism.